src/chromium/interfaces/launchBrowser.js

```diff
@@ -1,26 +1,37 @@
 import { getIsFirefoxInstalled } from "./getters.js";
-import { isCurrentTabValidUrlScheme } from "Shared/backgroundScripts/validTab.js";
 
 /**
  * Launches the Firefox variant. If Firefox is not installed, launch the Firefox download page.
  *
- * @param {*} tab The tab object to launch the browser with.
+ * @param {string} url The url to launch in the browser.
  * @param {boolean} usePrivateBrowsing  True if firefox should be launched in private browsing mode, false for normal mode.
  * @returns {boolean} True if the browser was launched, false otherwise.
  */
-export async function launchBrowser(tab, usePrivateBrowsing = false) {
+export async function launchBrowser(url, usePrivateBrowsing = false) {
+  if (
+    url &&
```

Comment:
```suggestion
    !url ||
```

Logically, an empty value should be rejected.
 (For readability, you may want to wrap the next part of the condition in parentheses)

This logic is also repeated in multiple files. It would be cleaner to declare a new utility function and call that instead of the current snippet.